LayoutTests/imported/w3c/resources/import-expectations.json

```
@@ -65,6 +65,7 @@
     "web-platform-tests/css/css-scrollbars": "import",
     "web-platform-tests/css/css-shadow-parts": "import",
     "web-platform-tests/css/css-shapes": "import",
+    "web-platform-tests/css/css-sizing": "import",
     "web-platform-tests/css/css-sizing/aspect-ratio": "import",
     "web-platform-tests/css/css-sizing/contain-intrinsic-size": "import",
```

```suggestion
    "web-platform-tests/css/css-sizing": "import",
```

The 2 lines are redundant as the style checker says.

---

Will update once EWS give me any other to fix. :-)

---

@nt1m - I missed 'css-sizing/' vs 'css-sizing' so I was only imported partial first but now they are all. Just tagging, if you want to re-review. I will address above and once all imports go through EWS once.

---

Importing web-platform-tests/css/css-sizing should import the subdirectories too. So all you need to do here is to remove these 2 lines:
```
    "web-platform-tests/css/css-sizing/aspect-ratio": "import",
    "web-platform-tests/css/css-sizing/contain-intrinsic-size": "import",
```

---

Done.